tests/test_ecephys/test_tools_spikeinterface.py

```diff
@@ -52,7 +50,7 @@ def setUp(self):
         )
 
     def test_default_values(self):
-        add_electrical_series(recording=self.test_recording_extractor, nwbfile=self.nwbfile, iterator_type=None)
+        add_electrical_series(recording=self.test_recording_extractor, nwbfile=self.nwbfile)
```

Comment:
This type of change does actually alter behavior of the tests

`iterator_type = None` means 'disable iteration completely', as in the `data` is not wrapped in an iterator at all

This is one of those behaviors that will be changed soon in the backend integration but for now, as a non-altering cleanup PR, I'd say revert that too

Reply:
ah, ok I didn't notice that. the typehints of `the iterator_type` arg of `add_electrical_series` indicate that the arg must be a string, so I thought this was a mistake. I'll just update those typehints instead.

Reply:
Yeah the annotations in the signature are not correct, those could be updated here